server/controllers/rescue.ctrl.js

```diff
@@ -15,4 +17,81 @@ module.exports = {
         res.send(err);
       });
   },
+
+  // Get rescue updates
+  getUpdates: (req, res) => {
+    const { id } = req.params;
+
+    const getUpdates = (rescue) => {
+      if (!isEmpty(rescue)) {
+        Update.find({ rescueId: rescue._id })
+          .then((updates) => res.status(200).send(updates))
```

Comment:
Might need to update this so updates are ordered by timestamps. I noticed in the staging version that the timestamps of other updates are arranged from newest to oldest.